### PR TITLE
findify analytics refactor, product card

### PR DESCRIPTION
**Pull request by AndreyPetlyashenko**

### CODE REVIEW

File: assets/findify-recommendation.css [hunk 1]
```diff
@@ -114,4 +109,27 @@
...
+.swiper-button-prev:after {
+    font-size: 10px !important;
+    color: white;
+    font-weight: 700;
+}
+
+.adapt {
+    width: 100%;
+    aspect-ratio: 1/1.5;
+    object-fit: cover;
```
Comment (wolff95) [inline assets/findify-recommendation.css:134]: .adapt class probably should be within product-card.css ?

File: snippets/findify-product-card.liquid [hunk 2]
```diff
@@ -1,37 +1,75 @@
 {%- if card_product and card_product != empty -%}
-  <div 
+  {% comment %}
```
Comment (wolff95) [inline snippets/findify-product-card.liquid:2]: I think we can remove this comment. If you wish to save this somewhere, maybe even drop it on a Slack channel and pin it.

One question, the <a ? navigation to product details?

File: assets/findify-recommendation.css (hunk 1 above)
Comment (AndreyPetlyashenko) [reply assets/findify-recommendation.css:134]: done)

File: snippets/findify-product-card.liquid (hunk 2 above)
Comment (AndreyPetlyashenko) [reply snippets/findify-product-card.liquid:2]: done))

**Pull request closed without merging**